### DISCUSSION TITLE
Unable to confirm Univeristy ID while signing for Azure Students

### DISCUSSION THREAD

**Discussion by Sunilshah-7** [microsoft/SATechnicalOnboarding#62] [Questions]
Hello, I have just started my onboarding process. Section 0: Prerequisites told me to create an Azure accounts for students so I went through this link: https://azure.microsoft.com/en-us/free/students/
but at the end I see this message. Because of this I cannot deploy my static web app to azure. How to solve this?
My email is Sunil.shah@studentambassadors.com
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/43769682/960cf866-7b49-4a1b-8033-bc632f032f6f)

**Comment by ManuSquall** [comment]
Hello @Sunilshah-7 , hope you're doing well.

With the student ambassador email `(xxx@studentambassadors.com)` you don't need to pass trought Azure for student Program; you could directly connect on Azure portal (https://portal.azure.com/) with the email.

Once connected if you don't have credit to create ressource you may consider to join Azure for student program with your personal email account eg:  `(xxx@outlook.com)`




🕯️If this answer helped you or resolved your issue kindly mark it as answer

**Reply by Sunilshah-7** [reply]
Thank you for your response. Does it mean that even if I have ambassador email (xxx@studentambassadors.com), I would have to get subscription in order to use Azure services?

**Reply by ManuSquall** [reply]
To answer the question: not necessarily

As i mentionned, i firstly advise you to connect on azure Portal (here : https://portal.azure.com/) with your ambassador email `(xxx@studentambassadors.com)`,

Then, If some credits have been allocated to your account you should see a notification on the top right like this:
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/29026887/273ca2b8-704b-47b4-acb3-8b045d416813)

**If it's not the case**, you should join and apply for the Azure for student program, **this time with your personal email acount**.

Let me know i you have other inquiries

🕯️If this answer helped you or resolved your issue kindly mark it as answer

**Reply by Sunilshah-7** [reply]
Thank you for your reply.
But Looks like I don't have any credits for Azure. Do you have any idea how can I get some credits for free or is there anything wrong with my account.
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/43769682/2eb17066-8262-4128-b854-0f79f9e9cab6)

**Reply by ManuSquall** [reply]
The lack of credit is probably due to your actual milestone which should be "New", according to this [doc](https://stdntpartners.sharepoint.com/sites/SAProgramHandbook/SitePages/Advancing-Milestones.aspx) it is from the Alpha Milestone that you begin to receive 150$ each month

You have to join the Azure for student program then! Please connect to it with your personnal email account

[Join the program by clicking here](https://signup.azure.com/studentverification?offerType=1&correlationId=c221403155d446e6af43bd7736e162c2)

🕯️If this answer helped you or resolved your issue kindly mark it as answer

**Reply by Sunilshah-7** [reply]
Oh, I think I got it. Thank you.

**Reply by ManuSquall** [reply]
Great, you're welcome; @Sunilshah-7 kindly mark the discussion as answered if you resolve your issue please

**Reply by haidarjomaa** [reply]
I am having the same problem after following all the procedures/links you mentioned in the thread. I'm still unable to validate even though my email is correct and valid

**Reply by AnthonyByansi** [reply]
Hey @haidarjomaa if you're not in position to redeem your Azure students credits, it might be that your institution domain is not among those listed. 

Alternatively you can redeem your Azure monthly credits [Here](https://my.visualstudio.com/). This is only possible if you advanced to Alpha

![Azure monthly](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/eb739443-cfa8-443c-88ec-4982f9355fb2)

**Reply by NandiniAg05** [reply]
I'm not able to sign up in my azure account. I have reached alpha milestone and want to redeem my credits, but it is asking me to fill a form and even after filling, it showing nothing. Please help me out with this.

**Reply by AnthonyByansi** [reply]
Hi @NandiniAg05 I'm not sure of the form you're filling. Have you looked at this solution 👉https://github.com/microsoft/SATechnicalOnboarding/discussions/62#discussioncomment-8168248. It might eventually be helpful.

Also please note that it might take some time to have your Azure credits processed. Please consider chatting with the SA-coordinator on teams
